engine/src/main/java/org/terasology/rendering/dag/StateChange.java

```diff
@@ -22,4 +22,6 @@
     T getValue();
```

Comment:
I just realised that we might not need this getValue() method anymore. That was necessary only if an external object generated the tasks. What do you think?